{
  error: Error: Queue "VerifyTicketsChatBotInactives" not found.
      at addJob (/Users/renatomendonca/Desktop/Node/serverwpp/src/libs/Queue.ts:44:9)
      at bullMQ (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:22:8)
      at bootstrap (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts:11:14)
      at application (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts:13:17)
      at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:4:7)
      at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)
      at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)
      at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js:69:33)
      at node:internal/modules/cjs/loader:1698:10
      at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js:114:20),
  level: 'error',
  message: 'uncaughtException: Queue "VerifyTicketsChatBotInactives" not found.\n' +
    'Error: Queue "VerifyTicketsChatBotInactives" not found.\n' +
    '    at addJob (/Users/renatomendonca/Desktop/Node/serverwpp/src/libs/Queue.ts:44:9)\n' +
    '    at bullMQ (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:22:8)\n' +
    '    at bootstrap (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts:11:14)\n' +
    '    at application (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts:13:17)\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:4:7)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1698:10\n' +
    '    at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js:114:20)',
  stack: 'Error: Queue "VerifyTicketsChatBotInactives" not found.\n' +
    '    at addJob (/Users/renatomendonca/Desktop/Node/serverwpp/src/libs/Queue.ts:44:9)\n' +
    '    at bullMQ (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:22:8)\n' +
    '    at bootstrap (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts:11:14)\n' +
    '    at application (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts:13:17)\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:4:7)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1698:10\n' +
    '    at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js:114:20)',
  exception: true,
  date: 'Wed Dec 04 2024 20:34:33 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 4889,
    uid: 501,
    gid: 20,
    cwd: '/Users/renatomendonca/Desktop/Node/serverwpp',
    execPath: '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
    version: 'v23.3.0',
    argv: [
      '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 82870272,
      heapTotal: 35307520,
      heapUsed: 14113552,
      external: 2397240,
      arrayBuffers: 65335
    }
  },
  os: {
    loadavg: [ 1.49658203125, 1.56591796875, 2.07421875 ],
    uptime: 88978
  },
  trace: [
    {
      column: 9,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/libs/Queue.ts',
      function: 'addJob',
      line: 44,
      method: null,
      native: false
    },
    {
      column: 8,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts',
      function: 'bullMQ',
      line: 22,
      method: null,
      native: false
    },
    {
      column: 14,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts',
      function: 'bootstrap',
      line: 11,
      method: null,
      native: false
    },
    {
      column: 17,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts',
      function: 'application',
      line: 13,
      method: null,
      native: false
    },
    {
      column: 7,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts',
      function: null,
      line: 4,
      method: null,
      native: false
    },
    {
      column: 14,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1546,
      method: null,
      native: false
    },
    {
      column: 25,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js',
      function: 'Module._compile',
      line: 568,
      method: '_compile',
      native: false
    },
    {
      column: 33,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js',
      function: 'Module.m._compile',
      line: 69,
      method: '_compile',
      native: false
    },
    {
      column: 10,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1698,
      method: null,
      native: false
    },
    {
      column: 20,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-6972605372296943.js',
      function: 'require.extensions..jsx.require.extensions..js',
      line: 114,
      method: '.js',
      native: false
    }
  ],
  timestamp: '2024-12-04T23:34:33.474Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:35:37.285Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:35:38.293Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:35:39.300Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:35:40.308Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:35:41.313Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:36:36.064Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:36:37.074Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:37:30.387Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:37:31.397Z'
}
{
  error: Error: Cannot find module '../app/config-env'
  Require stack:
  - /Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts
      at Function._resolveFilename (node:internal/modules/cjs/loader:1239:15)
      at Function._load (node:internal/modules/cjs/loader:1064:27)
      at TracingChannel.traceSync (node:diagnostics_channel:322:14)
      at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
      at Module.require (node:internal/modules/cjs/loader:1325:12)
      at require (node:internal/modules/helpers:136:16)
      at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:3:1)
      at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)
      at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)
      at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-05741314521180074.js:69:33) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [ '/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts' ]
  },
  level: 'error',
  message: "uncaughtException: Cannot find module '../app/config-env'\n" +
    'Require stack:\n' +
    '- /Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts\n' +
    "Error: Cannot find module '../app/config-env'\n" +
    'Require stack:\n' +
    '- /Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts\n' +
    '    at Function._resolveFilename (node:internal/modules/cjs/loader:1239:15)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1064:27)\n' +
    '    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n' +
    '    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)\n' +
    '    at Module.require (node:internal/modules/cjs/loader:1325:12)\n' +
    '    at require (node:internal/modules/helpers:136:16)\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:3:1)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-05741314521180074.js:69:33)',
  stack: "Error: Cannot find module '../app/config-env'\n" +
    'Require stack:\n' +
    '- /Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts\n' +
    '    at Function._resolveFilename (node:internal/modules/cjs/loader:1239:15)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1064:27)\n' +
    '    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n' +
    '    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)\n' +
    '    at Module.require (node:internal/modules/cjs/loader:1325:12)\n' +
    '    at require (node:internal/modules/helpers:136:16)\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:3:1)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-05741314521180074.js:69:33)',
  exception: true,
  date: 'Wed Dec 04 2024 20:37:49 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 5011,
    uid: 501,
    gid: 20,
    cwd: '/Users/renatomendonca/Desktop/Node/serverwpp',
    execPath: '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
    version: 'v23.3.0',
    argv: [
      '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 82100224,
      heapTotal: 33505280,
      heapUsed: 18089096,
      external: 2363328,
      arrayBuffers: 49867
    }
  },
  os: { loadavg: [ 1.359375, 1.466796875, 1.923828125 ], uptime: 89174 },
  trace: [
    {
      column: 15,
      file: 'node:internal/modules/cjs/loader',
      function: 'Function._resolveFilename',
      line: 1239,
      method: '_resolveFilename',
      native: false
    },
    {
      column: 27,
      file: 'node:internal/modules/cjs/loader',
      function: 'Function._load',
      line: 1064,
      method: '_load',
      native: false
    },
    {
      column: 14,
      file: 'node:diagnostics_channel',
      function: 'TracingChannel.traceSync',
      line: 322,
      method: 'traceSync',
      native: false
    },
    {
      column: 24,
      file: 'node:internal/modules/cjs/loader',
      function: 'wrapModuleLoad',
      line: 218,
      method: null,
      native: false
    },
    {
      column: 12,
      file: 'node:internal/modules/cjs/loader',
      function: 'Module.require',
      line: 1325,
      method: 'require',
      native: false
    },
    {
      column: 16,
      file: 'node:internal/modules/helpers',
      function: 'require',
      line: 136,
      method: null,
      native: false
    },
    {
      column: 1,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts',
      function: null,
      line: 3,
      method: null,
      native: false
    },
    {
      column: 14,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1546,
      method: null,
      native: false
    },
    {
      column: 25,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js',
      function: 'Module._compile',
      line: 568,
      method: '_compile',
      native: false
    },
    {
      column: 33,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-05741314521180074.js',
      function: 'Module.m._compile',
      line: 69,
      method: '_compile',
      native: false
    }
  ],
  timestamp: '2024-12-04T23:37:49.258Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:38:08.870Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:38:29.281Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:38:30.291Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-04T23:39:02.555Z'
}
{
  error: Error: ⨯ Unable to compile TypeScript:
  src/app/bull.ts(23,2): error TS1434: Unexpected keyword or identifier.
  src/app/bull.ts(23,13): error TS1434: Unexpected keyword or identifier.
  src/app/bull.ts(23,15): error TS1434: Unexpected keyword or identifier.
  src/app/bull.ts(24,2): error TS1005: 'while' expected.
  
      at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:1:7)
      at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)
      at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)
      at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:69:33)
      at node:internal/modules/cjs/loader:1698:10
      at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:114:20)
      at require.extensions.<computed> (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:71:20)
      at Object.nodeDevHook [as .ts] (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/ts-node-dev/lib/hook.js:63:13)
      at Module.load (node:internal/modules/cjs/loader:1303:32)
      at Function._load (node:internal/modules/cjs/loader:1117:12),
  level: 'error',
  message: 'uncaughtException: ⨯ Unable to compile TypeScript:\n' +
    'src/app/bull.ts(23,2): error TS1434: Unexpected keyword or identifier.\n' +
    'src/app/bull.ts(23,13): error TS1434: Unexpected keyword or identifier.\n' +
    'src/app/bull.ts(23,15): error TS1434: Unexpected keyword or identifier.\n' +
    "src/app/bull.ts(24,2): error TS1005: 'while' expected.\n" +
    '\n' +
    'Error: ⨯ Unable to compile TypeScript:\n' +
    'src/app/bull.ts(23,2): error TS1434: Unexpected keyword or identifier.\n' +
    'src/app/bull.ts(23,13): error TS1434: Unexpected keyword or identifier.\n' +
    'src/app/bull.ts(23,15): error TS1434: Unexpected keyword or identifier.\n' +
    "src/app/bull.ts(24,2): error TS1005: 'while' expected.\n" +
    '\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:1:7)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1698:10\n' +
    '    at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:114:20)\n' +
    '    at require.extensions.<computed> (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:71:20)\n' +
    '    at Object.nodeDevHook [as .ts] (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/ts-node-dev/lib/hook.js:63:13)\n' +
    '    at Module.load (node:internal/modules/cjs/loader:1303:32)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1117:12)',
  stack: 'Error: ⨯ Unable to compile TypeScript:\n' +
    'src/app/bull.ts(23,2): error TS1434: Unexpected keyword or identifier.\n' +
    'src/app/bull.ts(23,13): error TS1434: Unexpected keyword or identifier.\n' +
    'src/app/bull.ts(23,15): error TS1434: Unexpected keyword or identifier.\n' +
    "src/app/bull.ts(24,2): error TS1005: 'while' expected.\n" +
    '\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:1:7)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1698:10\n' +
    '    at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:114:20)\n' +
    '    at require.extensions.<computed> (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:71:20)\n' +
    '    at Object.nodeDevHook [as .ts] (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/ts-node-dev/lib/hook.js:63:13)\n' +
    '    at Module.load (node:internal/modules/cjs/loader:1303:32)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1117:12)',
  exception: true,
  date: 'Wed Dec 04 2024 20:43:07 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 5140,
    uid: 501,
    gid: 20,
    cwd: '/Users/renatomendonca/Desktop/Node/serverwpp',
    execPath: '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
    version: 'v23.3.0',
    argv: [
      '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 64356352,
      heapTotal: 18038784,
      heapUsed: 12914976,
      external: 2260280,
      arrayBuffers: 16839
    }
  },
  os: {
    loadavg: [ 1.97314453125, 1.650390625, 1.8544921875 ],
    uptime: 89492
  },
  trace: [
    {
      column: 7,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts',
      function: null,
      line: 1,
      method: null,
      native: false
    },
    {
      column: 14,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1546,
      method: null,
      native: false
    },
    {
      column: 25,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js',
      function: 'Module._compile',
      line: 568,
      method: '_compile',
      native: false
    },
    {
      column: 33,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js',
      function: 'Module.m._compile',
      line: 69,
      method: '_compile',
      native: false
    },
    {
      column: 10,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1698,
      method: null,
      native: false
    },
    {
      column: 20,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js',
      function: 'require.extensions..jsx.require.extensions..js',
      line: 114,
      method: '.js',
      native: false
    },
    {
      column: 20,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js',
      function: 'require.extensions.<computed>',
      line: 71,
      method: '<computed>',
      native: false
    },
    {
      column: 13,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/ts-node-dev/lib/hook.js',
      function: 'Object.nodeDevHook [as .ts]',
      line: 63,
      method: 'ts]',
      native: false
    },
    {
      column: 32,
      file: 'node:internal/modules/cjs/loader',
      function: 'Module.load',
      line: 1303,
      method: 'load',
      native: false
    },
    {
      column: 12,
      file: 'node:internal/modules/cjs/loader',
      function: 'Function._load',
      line: 1117,
      method: '_load',
      native: false
    }
  ],
  timestamp: '2024-12-04T23:43:07.049Z'
}
{
  error: ReferenceError: ExpressAdapter is not defined
      at bullMQ (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:24:24)
      at bootstrap (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts:11:14)
      at application (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts:13:17)
      at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:4:7)
      at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)
      at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)
      at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:69:33)
      at node:internal/modules/cjs/loader:1698:10
      at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:114:20)
      at require.extensions.<computed> (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:71:20),
  level: 'error',
  message: 'uncaughtException: ExpressAdapter is not defined\n' +
    'ReferenceError: ExpressAdapter is not defined\n' +
    '    at bullMQ (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:24:24)\n' +
    '    at bootstrap (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts:11:14)\n' +
    '    at application (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts:13:17)\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:4:7)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1698:10\n' +
    '    at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:114:20)\n' +
    '    at require.extensions.<computed> (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:71:20)',
  stack: 'ReferenceError: ExpressAdapter is not defined\n' +
    '    at bullMQ (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts:24:24)\n' +
    '    at bootstrap (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts:11:14)\n' +
    '    at application (/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts:13:17)\n' +
    '    at Object.<anonymous> (/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:4:7)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js:568:25)\n' +
    '    at Module.m._compile (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1698:10\n' +
    '    at require.extensions..jsx.require.extensions..js (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:114:20)\n' +
    '    at require.extensions.<computed> (/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js:71:20)',
  exception: true,
  date: 'Wed Dec 04 2024 20:43:08 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 5144,
    uid: 501,
    gid: 20,
    cwd: '/Users/renatomendonca/Desktop/Node/serverwpp',
    execPath: '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
    version: 'v23.3.0',
    argv: [
      '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 80609280,
      heapTotal: 32980992,
      heapUsed: 22204416,
      external: 2363328,
      arrayBuffers: 49867
    }
  },
  os: {
    loadavg: [ 1.97314453125, 1.650390625, 1.8544921875 ],
    uptime: 89493
  },
  trace: [
    {
      column: 24,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/app/bull.ts',
      function: 'bullMQ',
      line: 24,
      method: null,
      native: false
    },
    {
      column: 14,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/app/boot.ts',
      function: 'bootstrap',
      line: 11,
      method: null,
      native: false
    },
    {
      column: 17,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/app/index.ts',
      function: 'application',
      line: 13,
      method: null,
      native: false
    },
    {
      column: 7,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts',
      function: null,
      line: 4,
      method: null,
      native: false
    },
    {
      column: 14,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1546,
      method: null,
      native: false
    },
    {
      column: 25,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/node_modules/source-map-support/source-map-support.js',
      function: 'Module._compile',
      line: 568,
      method: '_compile',
      native: false
    },
    {
      column: 33,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js',
      function: 'Module.m._compile',
      line: 69,
      method: '_compile',
      native: false
    },
    {
      column: 10,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1698,
      method: null,
      native: false
    },
    {
      column: 20,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js',
      function: 'require.extensions..jsx.require.extensions..js',
      line: 114,
      method: '.js',
      native: false
    },
    {
      column: 20,
      file: '/private/var/folders/fl/vyfbwjv544d6tk51w30mh1p40000gn/T/ts-node-dev-hook-8961940637376198.js',
      function: 'require.extensions.<computed>',
      line: 71,
      method: '<computed>',
      native: false
    }
  ],
  timestamp: '2024-12-04T23:43:08.657Z'
}
{
  error: TypeError: Cannot read properties of undefined (reading 'start')
      at /Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:5:6
      at processTicksAndRejections (node:internal/process/task_queues:105:5),
  level: 'error',
  message: "uncaughtException: Cannot read properties of undefined (reading 'start')\n" +
    "TypeError: Cannot read properties of undefined (reading 'start')\n" +
    '    at /Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:5:6\n' +
    '    at processTicksAndRejections (node:internal/process/task_queues:105:5)',
  stack: "TypeError: Cannot read properties of undefined (reading 'start')\n" +
    '    at /Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts:5:6\n' +
    '    at processTicksAndRejections (node:internal/process/task_queues:105:5)',
  exception: true,
  date: 'Wed Dec 04 2024 20:57:18 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 5368,
    uid: 501,
    gid: 20,
    cwd: '/Users/renatomendonca/Desktop/Node/serverwpp',
    execPath: '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
    version: 'v23.3.0',
    argv: [
      '/Users/renatomendonca/.nvm/versions/node/v23.3.0/bin/node',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 82837504,
      heapTotal: 35438592,
      heapUsed: 15716120,
      external: 2363288,
      arrayBuffers: 49827
    }
  },
  os: {
    loadavg: [ 1.5576171875, 1.70556640625, 1.80517578125 ],
    uptime: 90343
  },
  trace: [
    {
      column: 6,
      file: '/Users/renatomendonca/Desktop/Node/serverwpp/src/server.ts',
      function: null,
      line: 5,
      method: null,
      native: false
    },
    {
      column: 5,
      file: 'node:internal/process/task_queues',
      function: 'processTicksAndRejections',
      line: 105,
      method: null,
      native: false
    }
  ],
  timestamp: '2024-12-04T23:57:18.789Z'
}
{
  code: 'ERR_SERVER_NOT_RUNNING',
  level: 'error',
  message: 'Erro ao encerrar o servidor HTTP: Server is not running.',
  stack: 'Error: Server is not running.\n' +
    '    at Server.close (node:net:2323:12)\n' +
    '    at Object.onceWrapper (node:events:627:28)\n' +
    '    at Server.emit (node:events:525:35)\n' +
    '    at emitCloseNT (node:net:2383:8)\n' +
    '    at processTicksAndRejections (node:internal/process/task_queues:89:21)',
  timestamp: '2024-12-05T00:02:44.542Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:17:40.629Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:17:40.634Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:17:40.691Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:17:40.692Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:17:40.803Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:17:40.806Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:17:40.973Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:17:40.974Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:17:41.191Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:17:41.193Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:17:41.460Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:17:41.461Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:17:41.771Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:17:41.772Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:18:08.405Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:18:08.409Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:18:08.466Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:18:08.468Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:18:08.575Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:18:08.576Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:18:08.731Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:18:08.733Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 127.0.0.1:6379',
  stack: 'Error: connect ECONNREFUSED 127.0.0.1:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T13:18:08.950Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 127.0.0.1:6379',
  level: 'error',
  timestamp: '2024-12-05T13:18:08.951Z'
}
{
  error: ReferenceError: modules is not defined
      at bootstrap (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\app\boot.ts:13:2)
      at processTicksAndRejections (node:internal/process/task_queues:105:5)
      at application (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\app\index.ts:14:2),
  level: 'error',
  message: 'uncaughtException: modules is not defined\n' +
    'ReferenceError: modules is not defined\n' +
    '    at bootstrap (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\boot.ts:13:2)\n' +
    '    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n' +
    '    at application (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\index.ts:14:2)',
  stack: 'ReferenceError: modules is not defined\n' +
    '    at bootstrap (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\boot.ts:13:2)\n' +
    '    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n' +
    '    at application (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\index.ts:14:2)',
  exception: true,
  date: 'Thu Dec 05 2024 10:40:34 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 19040,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 83759104,
      heapTotal: 58650624,
      heapUsed: 32317432,
      external: 2539158,
      arrayBuffers: 98656
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 703212.312 },
  trace: [
    {
      column: 2,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\boot.ts',
      function: 'bootstrap',
      line: 13,
      method: null,
      native: false
    },
    {
      column: 5,
      file: 'node:internal/process/task_queues',
      function: 'processTicksAndRejections',
      line: 105,
      method: null,
      native: false
    },
    {
      column: 2,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\index.ts',
      function: 'application',
      line: 14,
      method: null,
      native: false
    }
  ],
  timestamp: '2024-12-05T13:40:34.078Z'
}
{
  error: Error: Error
      at Timeout._onTimeout (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\routes\index.ts:10:15)
      at listOnTimeout (node:internal/timers:594:17)
      at processTimers (node:internal/timers:529:7),
  level: 'error',
  message: 'uncaughtException: Error\n' +
    'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  stack: 'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  exception: true,
  date: 'Thu Dec 05 2024 10:41:37 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 15340,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 70111232,
      heapTotal: 28504064,
      heapUsed: 24096104,
      external: 2517709,
      arrayBuffers: 58413
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 703275.656 },
  trace: [
    {
      column: 15,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts',
      function: 'Timeout._onTimeout',
      line: 10,
      method: '_onTimeout',
      native: false
    },
    {
      column: 17,
      file: 'node:internal/timers',
      function: 'listOnTimeout',
      line: 594,
      method: null,
      native: false
    },
    {
      column: 7,
      file: 'node:internal/timers',
      function: 'processTimers',
      line: 529,
      method: null,
      native: false
    }
  ],
  timestamp: '2024-12-05T13:41:37.431Z'
}
{
  error: Error: Error
      at Timeout._onTimeout (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\routes\index.ts:10:15)
      at listOnTimeout (node:internal/timers:594:17)
      at processTimers (node:internal/timers:529:7),
  level: 'error',
  message: 'uncaughtException: Error\n' +
    'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  stack: 'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  exception: true,
  date: 'Thu Dec 05 2024 10:42:13 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 8764,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 69988352,
      heapTotal: 27717632,
      heapUsed: 24628816,
      external: 2517709,
      arrayBuffers: 58413
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 703311.671 },
  trace: [
    {
      column: 15,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts',
      function: 'Timeout._onTimeout',
      line: 10,
      method: '_onTimeout',
      native: false
    },
    {
      column: 17,
      file: 'node:internal/timers',
      function: 'listOnTimeout',
      line: 594,
      method: null,
      native: false
    },
    {
      column: 7,
      file: 'node:internal/timers',
      function: 'processTimers',
      line: 529,
      method: null,
      native: false
    }
  ],
  timestamp: '2024-12-05T13:42:13.445Z'
}
{
  error: Error: Error
      at Timeout._onTimeout (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\routes\index.ts:10:15)
      at listOnTimeout (node:internal/timers:594:17)
      at processTimers (node:internal/timers:529:7),
  level: 'error',
  message: 'uncaughtException: Error\n' +
    'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  stack: 'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  exception: true,
  date: 'Thu Dec 05 2024 10:42:49 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 13352,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 111861760,
      heapTotal: 70029312,
      heapUsed: 41760336,
      external: 2554296,
      arrayBuffers: 94616
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 703347.953 },
  trace: [
    {
      column: 15,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts',
      function: 'Timeout._onTimeout',
      line: 10,
      method: '_onTimeout',
      native: false
    },
    {
      column: 17,
      file: 'node:internal/timers',
      function: 'listOnTimeout',
      line: 594,
      method: null,
      native: false
    },
    {
      column: 7,
      file: 'node:internal/timers',
      function: 'processTimers',
      line: 529,
      method: null,
      native: false
    }
  ],
  timestamp: '2024-12-05T13:42:49.726Z'
}
{
  error: Error: Error
      at Timeout._onTimeout (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\routes\index.ts:10:15)
      at listOnTimeout (node:internal/timers:594:17)
      at processTimers (node:internal/timers:529:7),
  level: 'error',
  message: 'uncaughtException: Error\n' +
    'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  stack: 'Error: Error\n' +
    '    at Timeout._onTimeout (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts:10:15)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  exception: true,
  date: 'Thu Dec 05 2024 10:43:15 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 11284,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 70352896,
      heapTotal: 28504064,
      heapUsed: 24272280,
      external: 2517709,
      arrayBuffers: 58413
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 703373.453 },
  trace: [
    {
      column: 15,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\routes\\index.ts',
      function: 'Timeout._onTimeout',
      line: 10,
      method: '_onTimeout',
      native: false
    },
    {
      column: 17,
      file: 'node:internal/timers',
      function: 'listOnTimeout',
      line: 594,
      method: null,
      native: false
    },
    {
      column: 7,
      file: 'node:internal/timers',
      function: 'processTimers',
      line: 529,
      method: null,
      native: false
    }
  ],
  timestamp: '2024-12-05T13:43:15.234Z'
}
{
  error: Error: Cannot find module '../database'
  Require stack:
  - C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\app\database.ts
  - C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\app\boot.ts
  - C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\app\index.ts
  - C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\server.ts
      at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
      at Function._load (node:internal/modules/cjs/loader:1075:27)
      at TracingChannel.traceSync (node:diagnostics_channel:315:14)
      at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
      at Module.require (node:internal/modules/cjs/loader:1340:12)
      at require (node:internal/modules/helpers:141:16)
      at Object.<anonymous> (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\app\database.ts:1:1)
      at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)
      at Module._compile (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\node_modules\source-map-support\source-map-support.js:568:25)
      at Module.m._compile (C:\Users\suporte2\AppData\Local\Temp\ts-node-dev-hook-6690354482041099.js:69:33) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\database.ts',
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\boot.ts',
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\index.ts',
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\server.ts'
    ]
  },
  level: 'error',
  message: "uncaughtException: Cannot find module '../database'\n" +
    'Require stack:\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\database.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\boot.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\index.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\server.ts\n' +
    "Error: Cannot find module '../database'\n" +
    'Require stack:\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\database.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\boot.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\index.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\server.ts\n' +
    '    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1075:27)\n' +
    '    at TracingChannel.traceSync (node:diagnostics_channel:315:14)\n' +
    '    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)\n' +
    '    at Module.require (node:internal/modules/cjs/loader:1340:12)\n' +
    '    at require (node:internal/modules/helpers:141:16)\n' +
    '    at Object.<anonymous> (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\database.ts:1:1)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js:568:25)\n' +
    '    at Module.m._compile (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-6690354482041099.js:69:33)',
  stack: "Error: Cannot find module '../database'\n" +
    'Require stack:\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\database.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\boot.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\index.ts\n' +
    '- C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\server.ts\n' +
    '    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1075:27)\n' +
    '    at TracingChannel.traceSync (node:diagnostics_channel:315:14)\n' +
    '    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)\n' +
    '    at Module.require (node:internal/modules/cjs/loader:1340:12)\n' +
    '    at require (node:internal/modules/helpers:141:16)\n' +
    '    at Object.<anonymous> (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\database.ts:1:1)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js:568:25)\n' +
    '    at Module.m._compile (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-6690354482041099.js:69:33)',
  exception: true,
  date: 'Thu Dec 05 2024 10:55:23 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 7692,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 110559232,
      heapTotal: 72044544,
      heapUsed: 44017400,
      external: 2386388,
      arrayBuffers: 134768
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 704101.515 },
  trace: [
    {
      column: 15,
      file: 'node:internal/modules/cjs/loader',
      function: 'Function._resolveFilename',
      line: 1249,
      method: '_resolveFilename',
      native: false
    },
    {
      column: 27,
      file: 'node:internal/modules/cjs/loader',
      function: 'Function._load',
      line: 1075,
      method: '_load',
      native: false
    },
    {
      column: 14,
      file: 'node:diagnostics_channel',
      function: 'TracingChannel.traceSync',
      line: 315,
      method: 'traceSync',
      native: false
    },
    {
      column: 24,
      file: 'node:internal/modules/cjs/loader',
      function: 'wrapModuleLoad',
      line: 218,
      method: null,
      native: false
    },
    {
      column: 12,
      file: 'node:internal/modules/cjs/loader',
      function: 'Module.require',
      line: 1340,
      method: 'require',
      native: false
    },
    {
      column: 16,
      file: 'node:internal/modules/helpers',
      function: 'require',
      line: 141,
      method: null,
      native: false
    },
    {
      column: 1,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\app\\database.ts',
      function: null,
      line: 1,
      method: null,
      native: false
    },
    {
      column: 14,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1546,
      method: null,
      native: false
    },
    {
      column: 25,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js',
      function: 'Module._compile',
      line: 568,
      method: '_compile',
      native: false
    },
    {
      column: 33,
      file: 'C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-6690354482041099.js',
      function: 'Module.m._compile',
      line: 69,
      method: '_compile',
      native: false
    }
  ],
  timestamp: '2024-12-05T13:55:23.291Z'
}
{
  error: C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\libs\Queue.ts:118
          await worker.run();
          ^^^^^
  
  SyntaxError: await is only valid in async functions and the top level bodies of modules
      at wrapSafe (node:internal/modules/cjs/loader:1469:18)
      at Module.<anonymous> (node:internal/modules/cjs/loader:1491:20)
      at Module._compile (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\node_modules\source-map-support\source-map-support.js:568:25)
      at Module.m._compile (C:\Users\suporte2\AppData\Local\Temp\ts-node-dev-hook-38908472646549996.js:69:33)
      at node:internal/modules/cjs/loader:1689:10
      at require.extensions..jsx.require.extensions..js (C:\Users\suporte2\AppData\Local\Temp\ts-node-dev-hook-38908472646549996.js:114:20)
      at require.extensions.<computed> (C:\Users\suporte2\AppData\Local\Temp\ts-node-dev-hook-38908472646549996.js:71:20)
      at Object.nodeDevHook [as .ts] (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\node_modules\ts-node-dev\lib\hook.js:63:13)
      at Module.load (node:internal/modules/cjs/loader:1318:32)
      at Function._load (node:internal/modules/cjs/loader:1128:12),
  level: 'error',
  message: 'uncaughtException: await is only valid in async functions and the top level bodies of modules\n' +
    'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts:118\n' +
    '        await worker.run();\n' +
    '        ^^^^^\n' +
    '\n' +
    'SyntaxError: await is only valid in async functions and the top level bodies of modules\n' +
    '    at wrapSafe (node:internal/modules/cjs/loader:1469:18)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1491:20)\n' +
    '    at Module._compile (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js:568:25)\n' +
    '    at Module.m._compile (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1689:10\n' +
    '    at require.extensions..jsx.require.extensions..js (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js:114:20)\n' +
    '    at require.extensions.<computed> (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js:71:20)\n' +
    '    at Object.nodeDevHook [as .ts] (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\ts-node-dev\\lib\\hook.js:63:13)\n' +
    '    at Module.load (node:internal/modules/cjs/loader:1318:32)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1128:12)',
  stack: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts:118\n' +
    '        await worker.run();\n' +
    '        ^^^^^\n' +
    '\n' +
    'SyntaxError: await is only valid in async functions and the top level bodies of modules\n' +
    '    at wrapSafe (node:internal/modules/cjs/loader:1469:18)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1491:20)\n' +
    '    at Module._compile (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js:568:25)\n' +
    '    at Module.m._compile (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1689:10\n' +
    '    at require.extensions..jsx.require.extensions..js (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js:114:20)\n' +
    '    at require.extensions.<computed> (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js:71:20)\n' +
    '    at Object.nodeDevHook [as .ts] (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\ts-node-dev\\lib\\hook.js:63:13)\n' +
    '    at Module.load (node:internal/modules/cjs/loader:1318:32)\n' +
    '    at Function._load (node:internal/modules/cjs/loader:1128:12)',
  exception: true,
  date: 'Thu Dec 05 2024 12:32:35 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 10436,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 72286208,
      heapTotal: 34611200,
      heapUsed: 16214584,
      external: 2431816,
      arrayBuffers: 123046
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 709934.125 },
  trace: [
    {
      column: 18,
      file: 'node:internal/modules/cjs/loader',
      function: 'wrapSafe',
      line: 1469,
      method: null,
      native: false
    },
    {
      column: 20,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1491,
      method: null,
      native: false
    },
    {
      column: 25,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js',
      function: 'Module._compile',
      line: 568,
      method: '_compile',
      native: false
    },
    {
      column: 33,
      file: 'C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js',
      function: 'Module.m._compile',
      line: 69,
      method: '_compile',
      native: false
    },
    {
      column: 10,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1689,
      method: null,
      native: false
    },
    {
      column: 20,
      file: 'C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js',
      function: 'require.extensions..jsx.require.extensions..js',
      line: 114,
      method: '.js',
      native: false
    },
    {
      column: 20,
      file: 'C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-38908472646549996.js',
      function: 'require.extensions.<computed>',
      line: 71,
      method: '<computed>',
      native: false
    },
    {
      column: 13,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\ts-node-dev\\lib\\hook.js',
      function: 'Object.nodeDevHook [as .ts]',
      line: 63,
      method: 'ts]',
      native: false
    },
    {
      column: 32,
      file: 'node:internal/modules/cjs/loader',
      function: 'Module.load',
      line: 1318,
      method: 'load',
      native: false
    },
    {
      column: 12,
      file: 'node:internal/modules/cjs/loader',
      function: 'Function._load',
      line: 1128,
      method: '_load',
      native: false
    }
  ],
  timestamp: '2024-12-05T15:32:35.949Z'
}
{
  error: ReferenceError: jon is not defined
      at C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\libs\Queue.ts:27:14
      at Array.map (<anonymous>)
      at Object.<anonymous> (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\src\libs\Queue.ts:25:43)
      at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)
      at Module._compile (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\node_modules\source-map-support\source-map-support.js:568:25)
      at Module.m._compile (C:\Users\suporte2\AppData\Local\Temp\ts-node-dev-hook-22181715424701776.js:69:33)
      at node:internal/modules/cjs/loader:1689:10
      at require.extensions..jsx.require.extensions..js (C:\Users\suporte2\AppData\Local\Temp\ts-node-dev-hook-22181715424701776.js:114:20)
      at require.extensions.<computed> (C:\Users\suporte2\AppData\Local\Temp\ts-node-dev-hook-22181715424701776.js:71:20)
      at Object.nodeDevHook [as .ts] (C:\Users\suporte2\Documents\Programacao\Nodejs\serverwpp\node_modules\ts-node-dev\lib\hook.js:63:13),
  level: 'error',
  message: 'uncaughtException: jon is not defined\n' +
    'ReferenceError: jon is not defined\n' +
    '    at C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts:27:14\n' +
    '    at Array.map (<anonymous>)\n' +
    '    at Object.<anonymous> (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts:25:43)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js:568:25)\n' +
    '    at Module.m._compile (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1689:10\n' +
    '    at require.extensions..jsx.require.extensions..js (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js:114:20)\n' +
    '    at require.extensions.<computed> (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js:71:20)\n' +
    '    at Object.nodeDevHook [as .ts] (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\ts-node-dev\\lib\\hook.js:63:13)',
  stack: 'ReferenceError: jon is not defined\n' +
    '    at C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts:27:14\n' +
    '    at Array.map (<anonymous>)\n' +
    '    at Object.<anonymous> (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts:25:43)\n' +
    '    at Module.<anonymous> (node:internal/modules/cjs/loader:1546:14)\n' +
    '    at Module._compile (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js:568:25)\n' +
    '    at Module.m._compile (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js:69:33)\n' +
    '    at node:internal/modules/cjs/loader:1689:10\n' +
    '    at require.extensions..jsx.require.extensions..js (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js:114:20)\n' +
    '    at require.extensions.<computed> (C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js:71:20)\n' +
    '    at Object.nodeDevHook [as .ts] (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\ts-node-dev\\lib\\hook.js:63:13)',
  exception: true,
  date: 'Thu Dec 05 2024 15:49:38 GMT-0300 (Horário Padrão de Brasília)',
  process: {
    pid: 14456,
    uid: null,
    gid: null,
    cwd: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp',
    execPath: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
    version: 'v22.11.0',
    argv: [
      'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\Port\\app\\node.exe',
      'src/server.ts'
    ],
    memoryUsage: {
      rss: 75698176,
      heapTotal: 35397632,
      heapUsed: 16830632,
      external: 2442718,
      arrayBuffers: 133578
    }
  },
  os: { loadavg: [ 0, 0, 0 ], uptime: 721757.078 },
  trace: [
    {
      column: 14,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts',
      function: null,
      line: 27,
      method: null,
      native: false
    },
    {
      column: null,
      file: null,
      function: 'Array.map',
      line: null,
      method: 'map',
      native: false
    },
    {
      column: 43,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\src\\libs\\Queue.ts',
      function: null,
      line: 25,
      method: null,
      native: false
    },
    {
      column: 14,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1546,
      method: null,
      native: false
    },
    {
      column: 25,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\source-map-support\\source-map-support.js',
      function: 'Module._compile',
      line: 568,
      method: '_compile',
      native: false
    },
    {
      column: 33,
      file: 'C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js',
      function: 'Module.m._compile',
      line: 69,
      method: '_compile',
      native: false
    },
    {
      column: 10,
      file: 'node:internal/modules/cjs/loader',
      function: null,
      line: 1689,
      method: null,
      native: false
    },
    {
      column: 20,
      file: 'C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js',
      function: 'require.extensions..jsx.require.extensions..js',
      line: 114,
      method: '.js',
      native: false
    },
    {
      column: 20,
      file: 'C:\\Users\\suporte2\\AppData\\Local\\Temp\\ts-node-dev-hook-22181715424701776.js',
      function: 'require.extensions.<computed>',
      line: 71,
      method: '<computed>',
      native: false
    },
    {
      column: 13,
      file: 'C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\ts-node-dev\\lib\\hook.js',
      function: 'Object.nodeDevHook [as .ts]',
      line: 63,
      method: 'ts]',
      native: false
    }
  ],
  timestamp: '2024-12-05T18:49:38.984Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:02:48.760Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:48.762Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:48.763Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:48.763Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:02:52.079Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:52.080Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:52.081Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:52.081Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:02:55.489Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:55.490Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:55.491Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:55.491Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:02:59.002Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:59.003Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:59.004Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:02:59.004Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:03:02.525Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:02.526Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:02.527Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:02.528Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:03:06.366Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:06.367Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:06.367Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:06.368Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:03:09.936Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:09.937Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:09.938Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:09.938Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:03:13.688Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:13.689Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:13.689Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:13.690Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:03:17.355Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:17.356Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:17.357Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:03:17.357Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:26.305Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:26.306Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:26.306Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:26.306Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:33.239Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:33.240Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:33.240Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:33.241Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:36.841Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:36.844Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:36.845Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:36.845Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:41.482Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:41.483Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:41.484Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:41.484Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:44.928Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:44.930Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:44.930Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:44.931Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:48.752Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:48.753Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:48.753Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:48.754Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:52.309Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:52.311Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:52.312Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:52.312Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:56.018Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:56.019Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:56.020Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:56.020Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:09:59.777Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:59.778Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:59.778Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:09:59.778Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:03.507Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:03.508Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:03.508Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:03.509Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:07.403Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:07.404Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:07.404Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:07.405Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:11.202Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:11.203Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:11.203Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:11.203Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:15.123Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:15.124Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:15.125Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:15.125Z'
}
{
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  level: 'error',
  message: 'connect ETIMEDOUT',
  stack: 'Error: connect ETIMEDOUT\n' +
    '    at Socket.<anonymous> (C:\\Users\\suporte2\\Documents\\Programacao\\Nodejs\\serverwpp\\node_modules\\ioredis\\built\\Redis.js:170:41)\n' +
    '    at Object.onceWrapper (node:events:632:28)\n' +
    '    at Socket.emit (node:events:518:28)\n' +
    '    at Socket._onTimeout (node:net:595:8)\n' +
    '    at listOnTimeout (node:internal/timers:594:17)\n' +
    '    at processTimers (node:internal/timers:529:7)',
  timestamp: '2024-12-05T19:10:25.849Z'
}
{
  message: 'Erro na conexão com o Redis: connect ETIMEDOUT',
  level: 'error',
  timestamp: '2024-12-05T19:10:25.850Z'
}
{
  message: 'Erro na conexão com o Redis: connect ETIMEDOUT',
  level: 'error',
  timestamp: '2024-12-05T19:10:25.850Z'
}
{
  message: 'Erro na conexão com o Redis: connect ETIMEDOUT',
  level: 'error',
  timestamp: '2024-12-05T19:10:25.850Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:29.875Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:29.876Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:29.877Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:29.877Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:33.964Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:33.965Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:33.966Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:33.966Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:38.067Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:38.068Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:38.068Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:38.068Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:42.243Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:42.244Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:42.245Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:42.245Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:46.459Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:46.461Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:46.461Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:46.462Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:50.741Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:50.742Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:50.742Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:50.743Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:54.980Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:54.982Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:54.982Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:54.983Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:10:59.229Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:59.231Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:59.231Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:10:59.232Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:11:03.853Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:11:03.854Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:11:03.855Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:11:03.855Z'
}
{
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '109.199.105.36',
  port: 6379,
  level: 'error',
  message: 'connect ECONNREFUSED 109.199.105.36:6379',
  stack: 'Error: connect ECONNREFUSED 109.199.105.36:6379\n' +
    '    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1610:16)',
  timestamp: '2024-12-05T19:11:08.349Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:11:08.351Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:11:08.352Z'
}
{
  message: 'Erro na conexão com o Redis: connect ECONNREFUSED 109.199.105.36:6379',
  level: 'error',
  timestamp: '2024-12-05T19:11:08.353Z'
}
